### PR TITLE
plugin/ocp_fid_c6h:Added the ocp C6h feature api

### PR DESCRIPTION
**Pull request by VigneshwaranSaravana**
Added the ocp C6h feature api.

Reviewed-by: Karthik Balan karthik.b82@samsung.com, Arunpandian J arun.j@samsung.com
Signed-off-by: Vigneshwaran Saravanan/SSD TS Dev /SSIR/Engineer/Samsung Electronics <s.vignesh@samsung.com>

### CODE REVIEW

File: plugins/ocp/ocp-nvme.c [hunk 1]
```diff
@@ -1960,6 +1960,144 @@ static int set_dssd_power_state_feature(int argc, char **argv, struct command *c
...
+///////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+/// plp_health_check_interval
+
+static int set_plp_health_check_interval(int argc, char **argv, struct command *cmd,
+                struct plugin *plugin)
+{
+    const char *desc = "Define Issue Set Feature command (FID : 0xC2) PLP Health Check Interval";
```
Comment (ikegami-t) [inline plugins/ocp/ocp-nvme.c:1972]: The indentations should be fixed and also please refer the commit messages below to check patch.
1. https://github.com/linux-nvme/nvme-cli/pull/2079#issuecomment-1757475751
2. https://github.com/linux-nvme/nvme-cli/pull/2087#issuecomment-1759421313

Comment (igaw) [reply plugins/ocp/ocp-nvme.c:1972]: Yes, you can use checkpatch.pl from the Linux project to figure out the obvious style problems:

```shell
$ ~/work/linux/scripts/checkpatch.pl -g HEAD 
WARNING: Missing commit description - Add an appropriate one

WARNING: added, moved or deleted file(s), does MAINTAINERS need updating?
#20: 
new file mode 100644

ERROR: trailing whitespace
#95: FILE: Documentation/nvme-ocp-set-plp-health-check-interval.txt:11:
+'nvme ocp set-plp-health-check-interval' <device> [--plp_health_interval=<plp_health_interval> | -p <plp_health_interval>] [--save | -s] [--no-uuid | -n] $

ERROR: code indent should use tabs where possible
#153: FILE: plugins/ocp/ocp-nvme.c:1970:
+                struct plugin *plugin)$

WARNING: please, no spaces at the start of a line
#153: FILE: plugins/ocp/ocp-nvme.c:1970:
+                struct plugin *plugin)$

WARNING: please, no spaces at the start of a line
#155: FILE: plugins/ocp/ocp-nvme.c:1972:
+    const char *desc = "Define Issue Set Feature command (FID : 0xC2) PLP Health Check Interval";$
```

FWIW, we just added a `checkpatch.pl` github workflow check to libnvme. I suppose we should also
add this to nvme-cli soon. I just want to see how it work in libnvme first.

Comment (VigneshwaranSaravana) [reply plugins/ocp/ocp-nvme.c]: Incorporated the review comments

Comment (ikegami-t) [reply plugins/ocp/ocp-nvme.c]: Still checkpatch.pl erorrs and warnings are remained as [head.log](https://github.com/linux-nvme/nvme-cli/files/13268814/head.log) and [head^.log](https://github.com/linux-nvme/nvme-cli/files/13268828/head.log).

**Comment by igaw** [comment]
Thanks!

**Pull request merged**